docs/api.md

```diff
@@ -565,3 +565,13 @@ Acknowledge the receipt of a reliable message from reliable resources after crea
 #### Raises:
 
 `VantiqException`
+
+### Vantiq.register_subscriber\_on\_close()
+
+Register a callback to be called when the subscriber is closed. Useful for the subscriber to know that there's no
+longer any subscriber active (for whatever reason).  This could happen due to issues at the server side or the
```

Comment:
might sound better with a "with" here....
```suggestion
longer any subscriber active (for whatever reason).  This could happen due to issues at the server side or with the
```